development/website_updating.html

```diff
@@ -98,12 +98,12 @@ <h2><a name="ExistingPage" id="ExistingPage"></a>Existing
 
               <p>The pages should be viewable in most browsers. If possible
               test you changes in as many as possible. Preferably at least
-              with IE and one other modern browser if you are on Windows. But
-              if you are on another platform testing with the "default"
-              browser is best. Normally viewing the page in outside of the a
-              web server context will only show you the content and part of
-              the footer. This is because the site uses Server Side Includes
-              (SSI) which most browsers will not process. To look at the
+              two modern browsers if you are on Windows. But if you are on another
+              platform testing with the "default" browser is best.
+              Normally viewing the page in outside of the a web server context
```

Comment:
"outside of the a web server context" should probably be "outside of a web server context"

Reply:
Ah, wasn't looking at unmodified text. 
Fixed